{"uuid": "97f1e29f-d266-4c0a-a345-f388c7560113", "children": ["b3937e93-eec1-4a74-88fa-f53f291a28c2", "d1830d14-3c56-4258-892b-4df38f7065d0"], "befores": [{"name": "login_token", "status": "skipped", "statusDetails": {"message": "Skipped: ❌ 登录失败，跳过后续测试\n", "trace": "  File \"D:\\桌面\\pytest_demo\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"D:\\桌面\\pytest_demo\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"D:\\桌面\\pytest_demo\\venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"D:\\桌面\\pytest_demo\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\桌面\\pytest_demo\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\桌面\\pytest_demo\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 930, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"D:\\桌面\\pytest_demo\\conftest.py\", line 28, in login_token\n    pytest.skip(\"❌ 登录失败，跳过后续测试\")\n    ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\桌面\\pytest_demo\\venv\\Lib\\site-packages\\_pytest\\outcomes.py\", line 159, in skip\n    raise Skipped(msg=reason, allow_module_level=allow_module_level)\n"}, "start": 1760703793480, "stop": 1760703794454}], "start": 1760703793480, "stop": 1760703798420}